src/dune_pkg/opamUrl0.mli

```diff
@@ -19,7 +19,10 @@ val is_local : t -> bool
    local file system or [`Git] if it's a git repository (remote or otherwise)
    or [`Tar] if it's an http server with a tar resource. If it's neither of
    those cases, it will error out. *)
-val local_or_git_or_tar_only : t -> Loc.t -> [ `Path of Path.t | `Git | `Tar ]
+val local_or_git_or_archive_only
```

Comment:
You need to update the documentation above, as it still uses the old names.

Reply:
You are right but I don't have one...

Reply:
I have renamed in to `classify`.